pvactools/lib/color_peptides51mer.py

```diff
@@ -0,0 +1,386 @@
+import pandas as pd
+from bs4 import BeautifulSoup
+import os
+import tempfile
+import xlsxwriter
+
+class AminoAcid:
+
+    def __init__(self, nucleotide, bold, color, underline, large, position, open_tag, close_tag):
+        self.nucleotide = nucleotide
+        self.bold = bold
+        self.color = color
+        self.underline = underline
+        self.large = large
+        self.position = position 
+        self.open_tag = open_tag
+        self.close_tag = close_tag
+
+    def view(self):
+        print("Nucleotide: ", self.nucleotide)
+        print("Open Tag: ", self.open_tag)
+        print("Close Tag: ", self.close_tag)
+        print("Bold: ", self.bold)
+        print("Color: ",self.color)
+        print("Underline: ", self.underline)
+        print("Large: ", self.large)
+
+
+def annotate_every_nucleotide(sequence, classI_peptide, classII_peptide, 
+                              classI_ic50, classI_percentile, classII_ic50, classII_percentile, 
+                              classI_transcript, classII_transcript, 
+                              cIIC50_threshold, cIpercentile_threshold, cIIIC50_threshold, cIIpercent_threshold, probPos):
+
+    peptide_sequence = [] # Create a list to hold all AA of the 51mer
+
+    # Make the sequence a list of AminoAcid objects
+    for i in range(len(sequence)):
+        new_AA = AminoAcid(sequence[i], False, False, False, False, -1, False, False)
+        
+        if len(probPos) > 0 and sequence[i] in probPos:
+            new_AA.large = True
+        
+        peptide_sequence.append(new_AA)
+
+    # CLASS I
+    positions = []
+    # Get the positions in the peptide_sequence where the classI is located
+    for i in range(len(peptide_sequence)):
```

Comment:
I think this could be simplified to something like:
```
sequence = [s.nucleotide for s in peptide_sequence]
start = sequence.index(classI_peptide)
positions = list(range(start, start + len(b)))
```